src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action=""
+      data-qa="big"
+    >
+      <div class="field field-big">
+        <input
+          data-qa="keypress"
+          name="search-big"
+          class="search search-big"
```

Comment:
```suggestion
          class="search"
```
є просто search і search з модифікатором small, великого немає, тому що якщо є великий, то де тоді той "нормальний" через який "великий" вважається великим